schedule/yam/agama_nvme.yaml

```diff
@@ -0,0 +1,43 @@
+---
+name: agama_auto
+description: >
+  Prepare url for agama.auto boot parameter, boot, perform auto-installation with agama.
+schedule:
+  - yam/agama/boot_agama
+  - yam/agama/agama_auto
+  - installation/grub_test
+  - installation/first_boot
+  - yam/validate/validate_product
+  - yam/validate/validate_first_user
+  - console/system_prepare
+  - console/nvme_checks
+  - console/validate_partition_table_via_blkid
+  - console/validate_blockdevices
+  - console/validate_free_space
+  - console/validate_read_write
+test_data:
```

Comment:
please don't use test_data in the same file, always use the setting for test data, it might help to future refactors.

Reply:
Updated.